helm_deploy/hmpps-tier-sqs-tool/templates/dlq-transfer-cronjob.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: dlq-transfer-cronjob
+spec:
+  schedule: "*/15 * * * *"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: dlq-transfer
+              image: quay.io/hmpps/dps-tools
+              args:
+                - /bin/sh
+                - -c
+                - curl http://hmpps-tier-sqs-tool/transfer
```

Comment:
This won't be the right url will it? its hmpps-tier-sqs-tool.hmpps-tier-dev.svc.cluster.local?

Reply:
This is copy/pasted from
https://github.com/ministryofjustice/probation-offender-search-indexer/blob/2e16a7152d10a5eac8432fe1b86c792b0a70d8e9/helm_deploy/probation-offender-search-indexer/templates/housekeeping-cronjob.yaml
and I admit I made that URL up. The one in probation-offender-search-indexer is not environment-specific though

Reply:
lets try it, it might try local first, but I know we don't have effective controls to stop us calling other namespaces. so we could accidentally call  prod from dev for example.

Reply:
not the end of the world in this specific example, but definitely a cause for concern more widely